lib/net.c

```diff
@@ -198,8 +204,15 @@ gatekeeper_init_network(void)
 	return ret;
 }
 
-void
-gatekeeper_free_network(void)
+int
+gatekeeper_free_network(struct net_config *net_conf)
 {
 	close_num_ports(num_ports);
```

Comment:
Now that we are using Lua, variable `num_ports` should be a member of `struct net_config`.